test/pj-rehearse-integration/candidate/ci-operator/jobs/super/duper/super-duper-ciop-cfg-change-presubmits.yaml

```diff
@@ -16,15 +16,14 @@ presubmits:
         - --artifact-dir=$(ARTIFACTS)
         - --give-pr-author-access-to-namespace=true
         - --target=[images]
-        - --target=[release:latest]
```

Comment:
This is surprising -- @smarterclayton ?

Reply:
It's expected after regeneration - the source file is https://github.com/openshift/ci-operator-prowgen/blob/master/test/pj-rehearse-integration/candidate/ci-operator/config/super/duper/super-duper-ciop-cfg-change.yaml - it does not have a `promotion` stanza so prowgen [does not generate this target](https://github.com/openshift/ci-operator-prowgen/blob/master/cmd/ci-operator-prowgen/main.go#L360), because such component[ would not promote ](https://github.com/openshift/ci-operator-prowgen/blob/master/pkg/promotion/promotion.go#L23)any images.